test/test_grid_search.py

```diff
@@ -0,0 +1,163 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+import fairlearn.grid_search as gs
```

Comment:
nit: Looks like you only ever use BinaryClassificationGridSearch, so you could just 
```
from fairlearn.grid_search import BinaryClassificationGridSearch
```
and not put gs.... everywhere because gs isn't very descriptive either.

Reply:
I'll leave this as-is for now. I think I would like to leave something of the 'namespace' intact, at least while I'm getting used to Python